beps/0012-metrics-service/README.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Backstage Metrics Service
+status: implementable
+authors:
+  - '@kurtaking'
+owners:
+  - '@kurtaking'
+project-areas:
+  - core-framework
+  - observability
```

Comment:
I know there was talk of creating an observability project area when the auditor project area was created. Is this something your still interested in @schultzp2020? We created the notification project area well after it's first BEP so we can probably push that out.